src/Umbraco.Web.UI/config/tinyMceConfig.config

```diff
@@ -5,204 +5,228 @@
   <commands>
     <command>
       <umbracoAlias>code</umbracoAlias>
+      <name>Code</name>
       <icon>images/editor/code.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="code">code</tinyMceCommand>
       <priority>1</priority>
     </command>
     <command>
       <umbracoAlias>codemirror</umbracoAlias>
+      <name>Code Mirror</name>
       <icon>images/editor/code.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="codemirror">codemirror</tinyMceCommand>
       <priority>1</priority>
     </command>
     <command>
       <umbracoAlias>removeformat</umbracoAlias>
+      <name>Remove Format</name>
       <icon>images/editor/removeformat.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="removeformat">removeformat</tinyMceCommand>
       <priority>2</priority>
     </command>
-
     <command>
-      <umbracoAlias>Undo</umbracoAlias>
+      <umbracoAlias>undo</umbracoAlias>
+      <name>Undo</name>
       <icon>images/editor/undo.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="undo">undo</tinyMceCommand>
       <priority>11</priority>
     </command>
     <command>
-      <umbracoAlias>Redo</umbracoAlias>
+      <umbracoAlias>redo</umbracoAlias>
+      <name>Redo</name>
       <icon>images/editor/redo.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="redo">redo</tinyMceCommand>
       <priority>12</priority>
     </command>
     <command>
-      <umbracoAlias>Cut</umbracoAlias>
+      <umbracoAlias>cut</umbracoAlias>
+      <name>Cut</name>
       <icon>images/editor/cut.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="cut">cut</tinyMceCommand>
       <priority>13</priority>
     </command>
     <command>
-      <umbracoAlias>Copy</umbracoAlias>
+      <umbracoAlias>copy</umbracoAlias>
+      <name>Copy</name>
       <icon>images/editor/copy.gif</icon>
       <tinyMceCommand value="" userInterface="false" frontendCommand="copy">copy</tinyMceCommand>
       <priority>14</priority>
     </command>
-
+    <command>
+      <umbracoAlias>paste</umbracoAlias>
+      <name>Paste</name>
+      <icon>images/editor/paste.gif</icon>
+      <tinyMceCommand value="" userInterface="false" frontendCommand="paste">paste</tinyMceCommand>
+      <priority>15</priority>
+    </command>

```

Comment:
I think we are missing the paste button? It has been in previously versions of Umbraco. Some editors might just copy-paste directly content from a website into the editor, which cause they might copy-paste inline markup + inline styling. The "paste" or "paste from word" button used to clean inline markup/styling and inserted text as raw text.